daily-challenges/2025-01/05/problem.md

```
@@ -0,0 +1,85 @@
+## Đề bài:
+
+Một gia đình chuẩn bị gói bánh chưng để đón Tết. Công thức cho mỗi chiếc bánh chưng như sau:
+
+- **200g gạo nếp**.
+- **50g đỗ xanh**.
+- **100g thịt lợn**.
+- **5g hành củ**.
+
+Hãy viết một hàm TypeScript để tính toán số lượng nguyên liệu cần thiết để gói một số lượng bánh chưng nhất định.
+
+### Yêu cầu:
```

Markdownlint (0.37.0)

Expected: 1; Actual: 0; Below
Headings should be surrounded by blank lines

(MD022, blanks-around-headings)